docs/machine-learning/resources/tasks.md

```diff
@@ -0,0 +1,46 @@
+# Machine Learning Tasks
+After understanding what you are hoping to achieve with your data, you will need to pick the right machine learning task for your situation. Below, you can find an explanation of different machine learning tasks and some common use cases. 
```

Comment:
This seems a bit abrupt if I land here from search and know nothing about ML.
"After understanding what you are hoping to achieve with your data," kind of implies a previous step. So you might wanna start generic about some of the steps involved with machine learning.

For example, "In machine learning, you first need to define what you want to achieve with your data. After that, you need to pick..."

Don't use below.

Reply:
Sure. I might actually change this to just say something more like here is a list of common machine learning tasks... (worded better). There will be a different doc that describes the learning pipeline steps.